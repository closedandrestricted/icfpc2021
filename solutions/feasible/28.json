{"vertices":[[22,10],[2,30],[2,30],[6,2]]}