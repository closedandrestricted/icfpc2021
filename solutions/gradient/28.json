{"vertices":[[10,31],[38,27],[14,3],[10,31]]}