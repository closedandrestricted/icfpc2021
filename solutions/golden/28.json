{"vertices":[[2,30],[6,2],[6,2],[2,30]]}